### ISSUE TITLE
[Bug] Cannot disable arrow keys as a keybind.

### ISSUE THREAD

**Issue by BrettplayMC** [ShareX/ShareX#4897]
The arrow keys have never been in my keybinds list as one but I found it on the website that they resize the shape. It seems that if there is no current shape, it captures my whole screen then lets me choose a shape.
Steps to reproduce:
Launch ShareX and push any arrow key, the 'gui' (per say to take a screenshot) will pop up.

Expected: Nothing happens
Actual: Screenshot is created
Heres a picture of my keybind settings:
![image](https://user-images.githubusercontent.com/20050432/85501305-b7be6f00-b5aa-11ea-8a13-4edb626f8ddb.png)

Here is the wiki I am referencing: https://getsharex.com/docs/region-capture

TL;DR: How do I disable arrow keys entirely so I can use them as normal?

**Comment by BrettplayMC** [comment]
Still a thing...
